### ISSUE TITLE
Consider query strings for context

### ISSUE THREAD

**Issue by pyphilia** [graasp/graasp-apps-query-client#83]
It would be really convenient to be able to set some value via querystrings such as language, view, mode, etc.
This would force the app in a particular context (it would be be prioritised compared to the parent's context).

**Comment by spaenleh** [comment]
I agree, but with the recent changes, this would have to either go in:
- graasp/ui: where the communication and the context is given to the app (inside the `AppItem` component)
- builder/player: where the `AppItem` is instantiated and the `contextPayload` is created. This was done to remove the need to pass a lot of props to the `AppItem` component.

But I guess we should extract as much logic from ui as possible.
